### PR TITLE
Fixed dispatch for updateCurrentAgentData

### PR DESCRIPTION
**Pull request by gabiwassan**
Hi team,
This PR added on menu-agents.js mapDispatchToProps to dispatch updateCurrentAgentData.
Plus: Prettier applied

### CODE REVIEW

File: public/components/common/welcome/components/menu-agent.js [hunk 1]
```diff
@@ -26,88 +27,165 @@ class WzMenuAgent extends Component {
       hoverAddFilter: '',
     };
 
-    this.menuAgent = window.localStorage.getItem('menuAgent') ? JSON.parse(window.localStorage.getItem('menuAgent')) : {};
+    this.menuAgent = window.localStorage.getItem('menuAgent')
+      ? JSON.parse(window.localStorage.getItem('menuAgent'))
+      : {};
 
     this.agentSections = {
```
Comment (matiasmoreno876) [inline public/components/common/welcome/components/menu-agent.js:34]: These sections could we get them from another side? to reduce the "lines" in the component?

Comment (gabiwassan) [reply public/components/common/welcome/components/menu-agent.js:34]: done!

**Review by matiasmoreno876 — APPROVED** [review]
LGTM!

**Review by Desvelao — APPROVED** [review]
LGTM!

**Pull request merged**